.gitignore

```diff
@@ -23,3 +23,5 @@ TAGS
 .vs
 .vscode
 .idea/
+# OSX trash
```

Comment:
I'd probably drop the comment, or mark it appropriately, like, _macOS folder attributes_.

Reply:
+1
I had the same comment, only in draft form because I failed to mail out this morning :-P

also I think these are roughly "alphabetized" so if you could move it up that would be great.

Reply:
Thank @venilnoronha , @alyssawilk  for comment! I'll close this PR!